packages/common/src/store/playback-position/slice.ts

```diff
@@ -31,22 +31,29 @@ const slice = createSlice({
       action: PayloadAction<InitializePlaybackPositionStatePayload>
     ) => {
       const { playbackPositionState } = action.payload
-      state.trackPositions =
-        playbackPositionState.trackPositions ?? state.trackPositions
+      const userIds = Object.keys(playbackPositionState)
+      userIds.forEach((userId) => {
+        state[userId] = playbackPositionState[userId]
+      })
```

Comment:
can we do `state = { ...playbackPositionState }` or does the proxy not like that?

Reply:
yea i tried that first, it doesn't like that unfortunately

Reply:
dang unlukcy